### PR TITLE
fix tag issue for openshift

### PR DESCRIPTION
**Pull request by praveenkumar**
We had a very long conversation here https://github.com/projectatomic/adb-utils/pull/104 please refer why we did this change.

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@praveenkumar This PR still does not handles the use case when an user users a version in the Vagrantfile, then logs in to the Vagrrantbox and does sccli stop and start OpenShift and user should get the same version.

Basically we should update openshift_options if we get something from environment.

**Comment by praveenkumar** [comment]
@LalatenduMohanty it does handle that. you can try it. we are updating openshift_options if we get something from environment - https://github.com/projectatomic/adb-utils/blob/master/utils/sccli.py#L137-L138

File: utils/sccli.py [hunk 1]
```diff
@@ -56,6 +54,19 @@ def system(cmd):
     returncode = ret.returncode
     return out, err, returncode
 
+def get_registry_image_tag_defaults():
+    try:
+        with open('/etc/sysconfig/openshift_option') as fh:
```
Comment (LalatenduMohanty) [inline utils/sccli.py]: We need to put `/etc/sysconfig/openshift_option` in to a variable and then use it in the script. We are already using it 4 places. If we don't have time for that then we need to create an issue and track it there.

**Comment by LalatenduMohanty** [comment]
@praveenkumar got it. It does handle that use case.

**Comment by LalatenduMohanty** [comment]
@praveenkumar LGTM

**Pull request merged**